pyvsystems/account.py

```diff
@@ -1,10 +1,10 @@
-from .setting import *
-from .crypto import *
-from .error import *
-from .words import WORDS
+from pyvsystems.setting import *
+from pyvsystems.crypto import *
+from pyvsystems.error import *
+from pyvsystems.words import WORDS
 from . import is_offline
 from . import default_chain
```

Comment:
If you prefer absolute import, please make all absolute. I find plenty of relative import